### PR TITLE
feat: k8s health check

### PR DESCRIPTION
**Pull request by jleach**
Add a health check I can use with kubernets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,9 +129,10 @@ The general command to deploy this to an OpenShift cluster is:
 
 ```bash
 helm template <RELEASE> ./devops/charts/controller
+-f ./devops/charts/controller/values_<ENVIRONMENT>.yaml
 --set-string tenant_id=<TENANT_ID> \
 --set-string tenant_api_key=<TENANT_API_KEY> \
---set-string redis_url=<REDIS_URL> \
+--set-string tenant_legacy_did=<TENANT_LEGACY_DID> \
```
Comment (bryce-mcmath) [inline README.md:135]: I just noticed I used `TRACTION_LEGACY_DID` in my PR that was merged this morning but here you have `TENANT_LEGACY_DID`. Which one is correct?

File: README.md [hunk 2]
```diff
@@ -145,10 +141,11 @@ And example command to deploy to the `e79518-dev` namespace is:
...
 ```bash
 helm template bcwallet ./devops/charts/controller
---set-string tenant_id=123-456-789 \
---set-string tenant_api_key=abc-def-ghi \
---set-string redis_url=redis://redis:6379/0 \
---set-file google_oauth_key.json=./google_oauth_key.json| \
+-f ./devops/charts/controller/values_dev.yaml
+--set-string tenant_id="$TRACTION_TENANT_ID"
+--set-string tenant_api_key="$TRACTION_TENANT_API_KEY"
+--set-string tenant_legacy_did="$TRACTION_DID"
```
Comment (bryce-mcmath) [inline README.md:147]: There's also `TRACTION_DID` being used here

**Review by bryce-mcmath — CHANGES_REQUESTED** [review]
Comment about env variable name

File: README.md [hunk 3]
```diff
@@ -145,10 +141,11 @@ And example command to deploy to the `e79518-dev` namespace is:
 
 ```bash
 helm template bcwallet ./devops/charts/controller
---set-string tenant_id=123-456-789 \
---set-string tenant_api_key=abc-def-ghi \
---set-string redis_url=redis://redis:6379/0 \
```
Comment (bryce-mcmath) [inline README.md:150]: We don't need a redis URL env variable anymore?

**Review by bryce-mcmath — APPROVED** [review]
One more question, otherwise LGTM

File: README.md (hunk 3 above)
Comment (jleach) [reply README.md:150]: No, I get it from a secret the ops team have already created. Still needed for local dev though.

**Pull request merged**